spec/waypoint_spec.js

```diff
@@ -1138,11 +1138,19 @@ describe('<Waypoint>', function() {
       this.parentStyle.height = 'auto';
       this.parentStyle.overflow = 'visible';
 
+      // This is only here to try and confuse the _findScrollableAncestor code.
+      document.body.style.overflow = 'auto';
+
       // Make the spacers large enough to make the Waypoint render off-screen
       this.topSpacerHeight = window.innerHeight + 1000;
       this.bottomSpacerHeight = 1000;
     });
 
+    afterEach(() => {
+      // Reset body style
+      document.body.style.overflow = '';
```

Comment:
This should probably reset it to whatever it was before Waypoint got involved, rather than an empty string